pvcompare/heat_pump_and_chiller.py

```diff
@@ -30,7 +30,7 @@ def calculate_cops_and_eers(
     Calculates the COPs of a heat pump or EERs of a chiller depending on `mode`.
 
     Temperature dependency is taken into consideration.
-    For these calculations the oemof.thermal `calc_cops()` functionality is
+    For these calculations the oemof.thermal :py:func:`~.calc_cops` functionality is
```

Comment:
```suggestion
    For these calculations the ``calc_cops()` <functionality>`_ functionality of `oemof.thermal <https://oemof-thermal.readthedocs.io/en/stable/>`_  is
```
You couldn't know, this time the function is outside pvcompare.